newsService/src/main/java/com/kovan/app/service/NewsService.java

```diff
@@ -37,26 +38,24 @@ public class NewsService {
     private final RestTemplate restTemplate;
     private final ObjectMapper objectMapper;
     private final NewsRepositoryService service;
```

Comment:
it should not service it should newsRepositoryService

Reply:
changed to newsRepositoryService